ui/partials/perf/graphsctrl.html

```diff
@@ -79,7 +79,16 @@
           <i class="text-warning fa fa-exclamation-circle"></i>
           <a href="perf.html#/alerts?id={{tooltipContent.alertSummary.id}}">
             Alert #{{tooltipContent.alertSummary.id}}</a>
-          <span class="text-muted">({{tooltipContent.alertSummary.getStatusText()}})</span>
+          <span class="text-muted">({{tooltipContent.alertSummary.getStatusText()}}
```

Comment:
Testing this, I actually found out this code was *wrong*. We should use the alert's status text, not the alert summary's. Try this instead:

```html
          <span class="text-muted text-nowrap">({{tooltipContent.alertSummary.alerts[0].getStatusText()}}
```